packages/l10n/src/create-l10n-injector/create-l10n-injector.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { reportErrorToSentry } from '@commercetools-frontend/sentry';
+import { getDisplayName } from '../utils';
+
+type InjectorOptions<T> = {
+  displayName: string;
+  propKey: string;
+  propLoadingKey: string;
+  loadLocale: (locale: string, cb: (error?: Error, data?: T) => void) => void;
+};
+
+export default function createL10NInjector<T>({
+  displayName,
+  propKey,
+  propLoadingKey,
+  loadLocale,
+}: InjectorOptions<T>) {
+  return function createHOC<U>(mapPropsToLocale: (props: any) => string) {
```

Comment:
Props is not any I think?

Reply:
<img width="868" alt="Screen Shot 2019-06-12 at 8 49 12 AM" src="https://user-images.githubusercontent.com/2425013/59329949-679ea280-8cf0-11e9-8770-1a2cb3d18680.png">